ocp_resources/limit_range.py

```diff
@@ -0,0 +1,26 @@
+# API reference: https://docs.openshift.com/container-platform/4.13/nodes/clusters/nodes-cluster-limit-ranges.html
+
+from ocp_resources.resource import NamespacedResource
+
+
+class LimitRange(NamespacedResource):
+    api_version = NamespacedResource.ApiVersion.V1
+
+    def __init__(
+        self,
+        limits=None,
+        **kwargs,
+    ):
+        """
+        Create LimitRange object.
+
+        Args:
+            limits (list): List of limits
```

Comment:
Can you add en example of the list that should be passed in the docstring?

Reply:
added example